### ISSUE TITLE
Previous conversations button disappears after navigating in secure mode

### ISSUE THREAD

**Issue by BinhTNT** [helpscout/beacon-android-sdk-sample#117]
Hello,
I have implemented secure mode to restore previous conversions for a user identity. I found an issue which after I open previous conversation screen, then go back by pressing back arrow, I notice that the Previous conversation button disappeared. I have to close and re-enter Beacon screen to see it again.

Could you please solve it?

* Beacon version: 2.1.2

**Steps to reproduce**
1. Open Beacon in secure mode
2. Open Previous conversations screen
3. Tap on back arrow in Previous conversations screen
4. Tap on Ask tab, notice that the Previous conversations button disappeared.

**Actual outcome**
Previous conversations button disappeared

**Expected outcome**
Previous conversations button should not be disappeared

Here is the video link of the issue: https://drive.google.com/file/d/1olcL3U4BDcGYkaGAh2SpHGdzKvm-v7Ec/view?usp=sharing

Thanks,
Binh

**Comment by jotaemepereira** [comment]
Hey @BinhTNT, thanks for taking the time to raise this issue. I did some testing on 2.1.2, and I wasn't able to replicate what you are seeing. I've added some questions to you so I can dig more in-depth:

1 - What are your Beacon settings on Help Scout, and are you using any of the [customizations](https://developer.helpscout.com/beacon-2/android/#settings-customization)?
2 - Did you turn on the `Don't keep activities` option from the developer options on Android?

Let me know, and I will continue investigating. Thanks.

**Comment by BinhTNT** [comment]
Hi @jotaemepereira Thank you for quick response.
For your questions:
1 - _What are your Beacon settings on Help Scout, and are you using any of the customizations?_
--> I'm not using those customizations in source code. We set them on Help Scout page.
2. _Did you turn on the Don't keep activities option from the developer options on Android?_
--> No, I did not turn it on. I checked again and it was turned off.

Here is the settings in Help Scout:

<img width="570" alt="Screen Shot 2020-08-20 at 12 38 48 PM" src="https://user-images.githubusercontent.com/45462932/90721174-6e587a80-e2e2-11ea-885a-bbda8499394d.png">

<img width="508" alt="Screen Shot 2020-08-20 at 12 38 55 PM" src="https://user-images.githubusercontent.com/45462932/90721182-70bad480-e2e2-11ea-8a31-2f8173a898e0.png">
